core/tx_pool.go

```diff
@@ -83,6 +84,12 @@ var (
 	// ErrKnownTransaction is returned if a transaction that is already in the pool
 	// attempting to be added to the pool.
 	ErrKnownTransaction = errors.New("known transaction")
+
+	// ErrBlacklistFrom is returned if a transaction's from/source address is blacklisted
+	ErrBlacklistFrom = errors.New("`from` address of transaction is blacklisted")
+
+	// ErrBlacklistTo is returned if a transaction's to/destination address is blacklisted
+	ErrBlacklistTo = errors.New("`to` address of transaction is blacklisted")
```

Comment:
what is the risk that this 'to' blacklist feature trying to mitigate? Was not clear why we need this in addition to 'from' blacklist

Reply:
One should not be able to send a transaction to a blacklisted account. Stopping someone from burning tokens.